x-pack/plugins/cases/public/containers/use_get_case.tsx

```diff
@@ -89,9 +94,12 @@ export const useGetCase = (caseId: string, subCaseId?: string): UseGetCase => {
         abortCtrlRef.current = new AbortController();
         dispatch({ type: 'FETCH_INIT', payload: { silent } });
 
-        const response = await (subCaseId
-          ? getSubCase(caseId, subCaseId, true, abortCtrlRef.current.signal)
-          : getCase(caseId, true, abortCtrlRef.current.signal));
+        const response: ResolvedCase = subCaseId
+          ? {
+              case: await getSubCase(caseId, subCaseId, true, abortCtrlRef.current.signal),
+              outcome: 'exactMatch',
```

Comment:
Might be worth adding a comment here stating why you are explicitly setting `exactMatch` here. Probably since the sub-cases won't have resolvedId's, but good to note if someone else works on it and is confused as to why we are explicitly setting the outcome

Reply:
Yes, we discussed that within the team, sub-cases are not used and the code will be probably deleted. I just put the things together in a way that makes sense, but this condition never happens indeed. I will add a comment to clarify that.

Reply:
Awesome, thanks!